src/lib/mina.ts

```diff
@@ -225,18 +226,26 @@ function LocalBlockchain({
         Math.ceil((new Date().valueOf() - startTime) / msPerSlot)
       );
     },
-    getAccount(publicKey: PublicKey): Account {
-      let ledgerAccount = ledger.getAccount(publicKey);
+    getAccount(
+      publicKey: PublicKey,
+      tokenId: string = getDefaultTokenId()
```

Comment:
maybe this is the place where we should do the conversion from `Field` to string? because here we know we need the actual value

Reply:
Good call! Made the `getAccount` accept a `Field` for the tokenId. Should the `Account` tokenId also be a Field or should that stay as a string? I changed the type to be a Field instead, wondering what's best here :/

Reply:
Yes `Field` on `Account ` feels right to me, because the other things on there are snarky types as well